### PR TITLE
Update monolog_email.rst

### PR DESCRIPTION
**Pull request by abdounikarim**
Hello,

I just add a link 😉

### CODE REVIEW

File: logging/monolog_email.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ How to Configure Monolog to Email Errors
 .. caution::
 
     This feature is not compatible yet with the new :doc:`Symfony mailer </mailer>`,
-    so it requires using SwiftMailer.
+    so it requires using :doc:`SwiftMailer </email>`.
```
Comment (OskarStark) [inline logging/monolog_email.rst]: We at Symfony want the new Mailer is used, so adding a link and make it more simple to go to SwiftMailer does not makes sense to me.

Comment (abdounikarim) [reply logging/monolog_email.rst]: @OskarStark actually, monolog isn't compatible with Mailer. That's why i add the link 😉

Comment (noniagriconomie) [reply logging/monolog_email.rst]: the PR has been merged recently
https://github.com/symfony/symfony/pull/33456
so IMO ne more need to promote swiftmailer email logging

exept for older version maybe :)

**Comment by HeahDude** [comment]
Closing here for the given reasons. Thanks

**Pull request closed without merging**